TOC/Proposal/voting-rules.md

```diff
@@ -0,0 +1,23 @@
+# Voting Rules for TOC Members
+
+This guideline is used for the decision-making process of TOC members in the TiDB community. Refer to Apache's voting rules, decisions are usually made by [lazy consensus](https://www.apache.org/foundation/glossary.html#LazyConsensus). If no consensus can be reached, the matter may be resolved by a majority vote. And potentially highly-controversial matters are recommended to be resolved by a majority vote, too.
+
+## Voting
+
+For all votes, voting must be open for at least 2 days. The deadline should be clearly stated in the call to vote. Votes are clearly indicated by a pull request adding an entry under the proposal folder.
+
+All TOC members have the right to make proposals. Only official members have the right to vote, and each has 1 vote. For public discussions, non-binding votes from the community are also highly appreciated.
+
+### Lazy Consensus
+
+The default decision-making mechanism in TiDB Community is [lazy consensus](https://www.apache.org/foundation/glossary.html#LazyConsensus). This means that any proposal is considered supported by the community as long as nobody objects.
+
+If any eligible voter raises objections, members eligible to vote will work together towards a solution that all involved can accept. This solution is again subject to lazy consensus.
+
+In case no consensus can be found, there will be a majority vote.
+
+### Majority Vote
+
+The majority votes must be called explicitly in a separate pull request. The subject must be prefixed with [VOTE]. In the body, the call to vote must state the proposal being voted on and the scope of the vote. The proposal that received the most votes in favor will be the result.
```

Comment:
```suggestion
The majority votes must be called explicitly in a separate pull request. In the body, the call to vote must state the proposal being voted on and the scope of the vote. The proposal that received the most votes in favor will be the result.
```

I think it is inherited from where Apache states for mailing list, it is not the case we collaborate on GitHub.

Reply:
Majority voting is initiated when lazy consensus fails, so I think it should be different from the previous proposal.

Reply:
Yes. But "The subject must be prefixed with [VOTE]." is obviously copy from Apache statements. If you are working on GitHub, there is no subject or "[VOTE]" which is an e-mail or e-mail alike concept...

Reply:
"The title of the pr must be prefixed with VOTE." Is it make sense?